.github/workflows/main.yml

```diff
@@ -74,6 +74,30 @@ jobs:
           retention-days: 1
           if-no-files-found: error
 
+  run-cwl-auto-conda:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: checkout-repository
+        uses: actions/checkout@v2
+
+      - name: install-basic-deps
+        uses: ./.github/actions/install-basic-deps
+
+      - name: run-workflow
+        shell: bash -l {0}
+        run: |
+          sudo apt-get update && sudo apt-get install --no-install-recommends --yes cwltool
```

Comment:
```suggestion
          sudo apt-get update && sudo apt-get install --no-install-recommends --yes python3-pip
          sudo python3 -m pip install cwltool[deps]
```

Reply:
The pipeline unfortunately still fails because the Conda python module is not found :/